README.md

```diff
@@ -14,7 +14,7 @@ steps:
     single-connection-string-name: <my connection string name for the single node>
     cluster-connection-string-name: <my connection string name for the cluster nodes>
     ravendb-license: <Single Line JSON License String>
-    ravendb-version: <RavenDB Version>
+    ravendb-version: <RavenDB Container Tag Name>
```

Comment:
Do you think this will be breaking or do we use the default value everywhere?

Reply:
We use the default value everywhere but I deliberately did not change the name of the variable. This change is just conveying the fact that version = tag.